onnx/common/status.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+namespace ONNX_NAMESPACE {
+namespace Common {
+
+enum StatusCategory {
+  NONE = 0,
+  OPSCHEMA = 1,
```

Comment:
CHECKER is more appropriate IMO, there are some checks don't belong to schema validation.